math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -2192,6 +2192,32 @@ RVec<T> Take(const RVec<T> &v, const int n)
    return r;
 }
 
+/// Return a copy of the container without the elements at the specified indices.
+///
+/// Duplicated and out-of-range indices in idxs are ignored.
+template <typename T>
+RVec<T> Drop(const RVec<T> &v, RVec<typename RVec<T>::size_type> idxs)
+{
+   RVec<T> r;
+   if (v.size() > idxs.size())
+      r.reserve(v.size() - idxs.size());
```

Comment:
Maybe move the call to `reserve()` after cleaning up `idxs`.

Reply:
makes sense, thanks